src/components/ConnectionBadge.tsx

```diff
@@ -0,0 +1,58 @@
+import { Platform, StyleSheet, useColorScheme, View } from 'react-native';
+import { DarkTheme, LightTheme } from '../constants/Colors';
+import Web3Text from './Web3Text';
+
+function ConnectionBadge() {
+  const isDarkMode = useColorScheme() === 'dark';
```

Comment:
You could probably combine these 2 into one hook i.e. `useTheme` that returns correct one, will simplify / reduce repetitiveness a bit as I assume you'll need it in many components

Reply:
great! i'll improve this in #45